.github/actions/verify-token/action.yml

```diff
@@ -13,6 +13,16 @@ inputs:
     type: string
     required: true
 
+  token:
+    description: "The GitHub Actions token."
+    required: false
+    default: ${{ github.token }}
+
+  output-predicate:
```

Comment:
Oh yes, this issue! https://github.com/slsa-framework/slsa-github-generator/issues/1480

> or would it be updating the predicate initially created by setup-token (I realize it's not now but eventually)?

Would be updating the one created by `setup-token`, if that's the route we go. However, it seems like the SLSA v1.0 provenance has less room for information than the token right now (in particular, there's no good place for the raw token inputs or the tool path: https://github.com/slsa-framework/slsa-github-generator/issues/1504). So I think on completion of the v1.0 spec, we can see if it would still apply.

Reply:
>  So I think on completion of the v1.0 spec, we can see if it would still apply.

SGTM!